src/storage/rpc.rs

```diff
@@ -107,10 +108,11 @@ impl DatabaseRef for RpcStorage {
             }
             let code = Bytecode::new_raw(code);
             let info = AccountInfo::new(balance, nonce, code.hash_slow(), code);
```

Comment:
Ah, it looks like the cleaner path is to construct `AccountBasic` & `EvmAccount` directly, instead of going through REVM types before converting.